server/routes/booking.py

```
@@ -0,0 +1,338 @@
+                return restful_response(
+                    status="error",
+                    message="Booking not found",
+                    status_code=404,
+                )
+            return restful_response(
+                status="success",
+                data=booking.to_dict(),
+                message="Booking retrieved successfully",
+                status_code=200,
+            )
+        except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)